.github/workflows/release.yml

```diff
@@ -1,56 +1,56 @@
-name: release
-
+name: Release - Linux
+env:
+  BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
+  IS_SNAPSHOT: ${{ github.ref_name != 'main' }}
 on:
   workflow_dispatch:
   push:
     tags:
-      - v* 
-    paths-ignore:      
-      - ".github/**"      
-      - "README.md"
+      - v*
```

Comment:
Triggering on tag push was somehow always a pain in MapLibre GL JS, because when stuff fails, the tags are still there and things get messy somehow.

I would recommend to trigger the release workflow only on `workflow_dispatch` and then the action can tag the git commit with a version. You can have a look how we do this in MapLibre GL JS at

https://github.com/maplibre/maplibre-gl-js/blob/11a1262601d9842efb4f6a44fed40fcb55fc2462/.github/workflows/release.yml#L34-L38